### PR TITLE
Update to iptables 1.8.7.

### PR DESCRIPTION
**Pull request by LaakkonenJussi**
Update iptables to 1.8.7. Drop some unavailable man pages from docs
package.

### CODE REVIEW

File: rpm/iptables.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Name:       iptables
 Summary:    Tools for managing Linux kernel packet filtering capabilities
-Version:    1.8.2
+Version:    1.8.7
 Release:    1
 Group:      System/Base
```
Comment (Tomin1) [inline rpm/iptables.spec]: You could drop all Group tags. They are [deprecated](https://sailfishos.org/wiki/Software_Packaging#RPM_packaging_Guidelines).

**Review by Tomin1 — COMMENTED** [review]
Some minor comments, otherwise looks good. 

You should update the mirror url to point to [sailfishos-mirror](https://github.com/sailfishos-mirror/) project.

File: rpm/iptables.spec (hunk 1 above)
Comment (LaakkonenJussi) [reply rpm/iptables.spec]: Right, cleaned up that and force pushed as minor change.

**Comment by LaakkonenJussi** [comment]
> Some minor comments, otherwise looks good.
> 
> You should update the mirror url to point to [sailfishos-mirror](https://github.com/sailfishos-mirror/) project.

Apparently we did not do a mass change for all the projects then? Changed that as well.

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [submodule "iptables"]
 	path = iptables
-	url = https://git.sailfishos.org/mirror/iptables.git
+	url = https://github.com/sailfishos/iptables.git
```
Comment (Tomin1) [inline .gitmodules]: sailfishos-mirror, not sailfishos

Comment (LaakkonenJussi) [reply .gitmodules]: Yep. Done.

**Review by Tomin1 — APPROVED** [review]
LGTM. :+1:

File: rpm/iptables.spec [hunk 3]
```diff
@@ -1,8 +1,7 @@
 Name:       iptables
 Summary:    Tools for managing Linux kernel packet filtering capabilities
-Version:    1.8.2
+Version:    1.8.7
 Release:    1
-Group:      System/Base
 License:    GPLv2
 URL:        http://www.netfilter.org/
 Source0:    http://www.netfilter.org/projects/iptables/files/%{name}-%{version}.tar.bz2
```
Comment (saukko) [inline rpm/iptables.spec]: This should be just %{name}-%{version}.tar.bz2 as we do not use the upstream tarballs.

File: rpm/iptables.spec [hunk 4]
```diff
@@ -1,8 +1,7 @@
 Name:       iptables
 Summary:    Tools for managing Linux kernel packet filtering capabilities
-Version:    1.8.2
+Version:    1.8.7
 Release:    1
-Group:      System/Base
 License:    GPLv2
 URL:        http://www.netfilter.org/
```
Comment (okodron) [inline rpm/iptables.spec]:
```suggestion
URL:        https://www.netfilter.org/projects/iptables/
```

File: rpm/iptables.spec [hunk 5]
```diff
@@ -47,7 +44,6 @@ network and you are using ipv6.
 
 %package doc
 Summary:   Documentation for %{name}
-Group:     Documentation
```
Comment (okodron) [inline rpm/iptables.spec:50]: `doc` sub-package can be marked with `BuildArch: noarch`.

**Review by okodron — COMMENTED** [review]
`%autosetup`, `%configure` and `%make_build` macros can be used here.
`rm -rf %{buildroot}` and explicit `rm -f %{buildroot}/%{_lib}/*.la` are not necessary any more.

File: rpm/iptables.spec (hunk 3 above)
Comment (LaakkonenJussi) [reply rpm/iptables.spec]: Changed in 4d5cb7d932a61067afa064f44ac0d7d118ba5690

File: rpm/iptables.spec (hunk 4 above)
Comment (LaakkonenJussi) [reply rpm/iptables.spec]: Added with 4d5cb7d932a61067afa064f44ac0d7d118ba5690

File: rpm/iptables.spec (hunk 5 above)
Comment (LaakkonenJussi) [reply rpm/iptables.spec:50]: Yep, added with 4d5cb7d932a61067afa064f44ac0d7d118ba5690

**Comment by LaakkonenJussi** [comment]
> `%autosetup`, `%configure` and `%make_build` macros can be used here.
> `rm -rf %{buildroot}` and explicit `rm -f %{buildroot}/%{_lib}/*.la` are not necessary any more.

Right. I also checked a bit on [how Fedora does it](https://src.fedoraproject.org/rpms/iptables/blob/rawhide/f/iptables.spec). Changed to use the newer macros.

**Review by okodron — APPROVED** [review]

**Pull request merged**